package.json

```diff
@@ -57,7 +57,7 @@
     "gatsby": "^4.9.2",
     "gatsby-image": "^3.7.1",
     "gatsby-plugin-image": "^2.9.0",
-    "gatsby-theme-carbon": "^3.0.0-next.13",
+    "gatsby-theme-carbon": "^3.1.1",
```

Comment:
I think `gatsby` will need updated as well to the latest 4.x minor, there's a mention of this in the build log